src/_gettsim/demographic_vars.py

```diff
@@ -105,8 +106,26 @@ def kind_bis_15(alter: int, kind: bool) -> bool:
     return out
 
 
-def kind_ab_7_bis_13(alter: int, kind: bool) -> bool:
-    """Calculate if child between 7 and 13 years old.
+def kind_bis_17(alter: int, kind: bool) -> bool:
+    """Calculate if underage person.
+
+    Parameters
+    ----------
+    alter
+        See basic input variable :ref:`alter <alter>`.
+    kind
+        See basic input variable :ref:`kind <kind>`.
+
+    Returns
+    -------
+
+    """
+    out = kind and (alter < 18)
```

Comment:
```suggestion
    out = kind and (alter <= 17)
```
Easier to match with function name.